cluster/types.go

```diff
@@ -199,7 +199,7 @@ type ResourceModel struct {
 // ResourceModelRange describes the detail of each modeling quota that ranges from min to max.
 // Please pay attention, by default, the value of min can be inclusive, and the value of max cannot be inclusive.
 // E.g. in an interval, min = 2, max =10 is set, which means the interval [2,10).
-// This rule ensure that all intervals have the same meaning. If the last interval is +8,
+// This rule ensure that all intervals have the same meaning. If the last interval is +��,
```

Comment:
Seems like the file encoding format is break, and maybe it do not need to change?

Reply:
fixed both of them. Thanks.